### PR TITLE
Makefile: Use speedb libs in `build_size` target

### LINKED ISSUE #275: Makefile: build_size target in the makefile is not adjusted to speedb libs
theres a new makefile target after the 7.7.3 rebase called build_size: 
which still uses the old lib names before #64  (librocksdb.a)

### PR DESCRIPTION
**Pull request by AmnonHanuhov**
Fixes #275

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2382,31 +2382,37 @@ build_folly:
...
+
 build_size:
 	# === normal build, static ===
 	$(MAKE) clean
 	$(MAKE) static_lib
-	$(REPORT_BUILD_STATISTIC) rocksdb.build_size.static_lib $$(stat --printf="%s" librocksdb.a)
-	strip librocksdb.a
-	$(REPORT_BUILD_STATISTIC) rocksdb.build_size.static_lib_stripped $$(stat --printf="%s" librocksdb.a)
+	$(REPORT_BUILD_STATISTIC) speedb.build_size.static_lib $$(stat --printf="%s" libspeedb.a)
+	strip -x libspeedb.a
```
Comment (mrambacher) [inline Makefile:2396]: Use $(PROJECT_NAME) instead of speedb

File: Makefile [hunk 2]
```diff
@@ -2382,31 +2382,37 @@ build_folly:
 
 REPORT_BUILD_STATISTIC?=echo STATISTIC:
 
+ifeq ($(PLATFORM), OS_MACOSX)
+LIB_EXTENSION = dylib
+else # Unix
+LIB_EXTENSION = so
+endif
+
```
Comment (mrambacher) [inline Makefile:2390]: There is already a $(PLATFORM_SHARED_EXT) that should be used

**Review by mrambacher — DISMISSED** [review]
This looks like it works most of the time.  However, I think if you said DEBUG_LEVEL=2 make build_size, it might fail.  Might want to check that just to be safe.

Otherwise, looks good!

**Comment by Yuval-Ariel** [comment]
@AmnonHanuhov , do you want to address marks last comment before i merge this?

**Comment by AmnonHanuhov** [comment]
> @AmnonHanuhov , do you want to address marks last comment before i merge this?

Yes, I will look into it today.

**Comment by AmnonHanuhov** [comment]
> 

Didn't seem to fail for me.

**Review by mrambacher — APPROVED** [review]
LGTM!

**Pull request merged**